{"name": "Unsuccessful login", "status": "passed", "steps": [{"name": "Error message is visible", "status": "passed", "start": 1737700077546, "stop": 1737700077546}], "parameters": [{"name": "username", "value": "'standard_use'"}, {"name": "password", "value": "'secret_sauce'"}, {"name": "driver", "value": "'Edge'"}], "start": 1737700077133, "stop": 1737700077582, "uuid": "57bf8168-80f3-445e-ae06-aa214133b653", "historyId": "86bc5338a5c26c00a62dcfde57e10d39", "testCaseId": "b876685be5aab7257716ff999cc789c0", "fullName": "Tests.test_Login.TestLogin#test_1", "labels": [{"name": "severity", "value": "normal"}, {"name": "feature", "value": "Login"}, {"name": "parentSuite", "value": "Tests"}, {"name": "suite", "value": "test_Login"}, {"name": "subSuite", "value": "TestLogin"}, {"name": "host", "value": "fv-az1704-590"}, {"name": "thread", "value": "1930-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "Tests.test_Login"}]}